timeseries/src/autogluon/timeseries/models/autogluon_tabular/tabular_model.py

```diff
@@ -51,10 +52,10 @@ class AutoGluonTabularModel(AbstractTimeSeriesModel):
     PREDICTION_BATCH_SIZE = 100_000
 
     TIMESERIES_METRIC_TO_TABULAR_METRIC = {
-        "MASE": "root_mean_squared_error",
+        "MASE": "mean_absolute_error",
```

Comment:
Any reason why we change the mapping?

Reply:
The new mapping consistently leads to better MASE/wQL loss. Training with the RMSE/MSE objective estimates the mean, while MASE is minimized by the median (assuming the data is scaled), so MAE makes more sense here.